torch/_dynamo/allowed_functions.py

```diff
@@ -276,7 +276,29 @@ def _builtin_constant_ids():
     return rv
 
 
+# A subcheck of is_allowed, we utilize this for patching is_allowed around distributed.
+# We do this because we want to allow these to be traced, and hence covered in skipfiles, but we do not want them to
+# become TorchVariable
+def _is_allowed_distributed(obj):
+    if not torch.distributed.is_available():
+        return True
```

Comment:
whats the use case for True here vs False?

Reply:
True just keeps it going where it used to before. This should always be safe.

Reply:
ok, might be cleaner to do the is_available check outside of this function (at least reads a little cleaner) but its equivalent